pkgs/development/web/nhost/default.nix

```diff
@@ -0,0 +1,30 @@
+{
+  buildGoModule
+, fetchFromGitHub
+, lib
+}:
+
+buildGoModule rec {
+  pname = "nhost";
+  version = "1.5.2";
+
+  src = fetchFromGitHub {
+    owner = "nhost";
+    repo = "cli";
+    rev = "v${version}";
+    sha256 = "sha256-mjhNZwbYc5RKxDBMmXCllKhHNJYwau5rQABUjKYIkf0=";
```

Comment:
Prefer `hash = ...` to `sha256 = ...` Both currently work, but the former will make it easier if the algorithm ever needs to change.

Since there could be different hashing algorithms in use, this make sense.